### ISSUE TITLE
Compatibility with AMD GPUs

### ISSUE THREAD

**Issue by svandenhaute** [openmm/openmm-torch#111]
Does this plugin work with AMD GPUs? PyTorch+ROCm is not installable via `conda`...

**Comment by peastman** [comment]
It includes an OpenCL implementation which works with AMD GPUs.  However, I believe the pytorch model will get executed on the CPU and only the OpenMM calculations will run on the GPU.

**Comment by svandenhaute** [comment]
Right, but given that the pytorch evaluation constitutes most of the calculation time (at least for relatively small systems and complex torch models), this would turn out to be rather slow, no?
